src/pages/AttendanceCheck.tsx

```diff
@@ -0,0 +1,288 @@
+import dayjs from "dayjs"
+import "dayjs/locale/ko"
+import { ChevronLeftIcon, ChevronRightIcon, UserGroupIcon } from "@heroicons/react/24/outline"
+import { PlusIcon, MinusIcon } from "@heroicons/react/24/solid"
+import { useState, useEffect } from "react"
+import { useNavigate } from "react-router-dom"
+
+import { Select, SelectContent, SelectGroup, SelectItem, SelectTrigger, SelectValue } from "../components/ui/select"
+import { Checkbox } from "../components/ui/checkbox"
+import { Button } from "../components/ui/button"
+import { McyMember } from "../types/McyMember"
+import { getMcyMemberApi } from "../api/mcyMemberApi"
+import Layout from "../components/Layout/Layout"
+import { CheckedMember } from "../types/CheckedMember"
+import { getAttendanceApi, updateAttendanceApi } from "../api/mcyAttendanceDataApi"
+import PAGE_PATH from "../constants/path"
+
+const AttendanceCheck = () => {
+  const navigate = useNavigate()
+  const [totalMember, setTotalMember] = useState<number>(0)
+  const [attendanceMember, setAttendanceMember] = useState<number>(0)
+  const [adultCount, setAdultCount] = useState<number>(0)
+  const [memberList, setMemberList] = useState<McyMember[]>([])
```

Comment:
```suggestion
  const [totalMember, setTotalMember] = useState<number>(0)
  const [attendanceMember, setAttendanceMember] = useState<number>(0)
  const [adultCount, setAdultCount] = useState<number>(0)
  const [memberList, setMemberList] = useState<McyMember[]>([])
```
이건 꼭 적용해야하는 부분은 아닌데, useState가 너무 많아져서 관리하고 변경하는게 복잡해 지는 경우에
객체 형태로 관리하는 것도 방법임.